doc/classes/PCKPacker.xml

```diff
@@ -42,6 +44,7 @@
 			<argument index="1" name="alignment" type="int">
```

Comment:
I'm not sure how to document the `alignment` parameter in a beginner-friendly way. I'm not even really sure how to use it myself. Help is welcome here :slightly_smiling_face: